src/stylesheets/components/_navigation.scss

```diff
@@ -38,7 +38,9 @@
 
 .usa-nav {
   $sliding-panel-width: 26rem;
+
   @include position(fixed, 0 0 0 auto);
+
```

Comment:
There should not be spaces here per the 18F front end guide which this project follows: https://frontend.18f.gov/#order @msecret @toolness